modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/writer/BatchEngineTaskItemWriter.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface BatchEngineTaskItemWriter extends Closeable {
 
+	public boolean canWrite();
```

Comment:
@ivicac what is this for? I will have to see an usage to understand what you are trying to do here.

Reply:
Basically the name implies, we could have a writer that can not write? Then why do we create it in the first place? It should be rejected on creation.

Reply:
@shuyangzhou that will be used in export part to trying to avoid writing at if there is no conditions(CSV/XLS will not have conditions if fieldNames are empty)

Reply:
Then we should not allow such writer being created, right? What's the point to have such non-writeable writer?